### ISSUE TITLE
after crash in dcs ssa works incorrectly

### ISSUE THREAD

**Issue by SchmidtDawid** [SimShaker-for-Aviators/SimShaker-for-Aviators-Releases#18]
**Describe the bug**
After being crashed of shoot down in DCS, shaker is working wrokg. Strong shake is turned on all time. Additionally if i turn off and on my shaker pad - HF8, then something still works wrong, and i get too big shakes. I need to turn on and off SSA. 

**To Reproduce**
Go to DCS, 
let shoot down youself

**Expected behavior**
works normally after mission restart

**System (please complete the following information):**
 - Windows version: win11
 - Simulator version: newest
 - SSA version: newest

**Comment by petsild** [comment]
Hi,
I have DCS constant vibrations after crashes or catapult.
I am using HF8 and SimShaker for Aviators 2.6.8 beta.

**Comment by f4l0** [comment]
Yeah, that is a known issue. It's sometimes hard to detect if crashed... I revisit the code and look for some improvements

**Comment by petsild** [comment]
Hi,
I'm trying the latest stable 2.7.0 and it still happens.
thanks for support.

**Comment by f4l0** [comment]
Still a known bug... I try to get some infos from ED and the beta tester group.

**Comment by petsild** [comment]
I'm happy to help test.

**Comment by petsild** [comment]
V 2.7.11 the continuous vibration will be felt when the machine touches the ground again after being repaired by the ground crew.
